### PR TITLE
Rename 'remap' variables

### PR DESCRIPTION
**Pull request by class4kayaker**
Fix for #1503.

### CODE REVIEW

**Comment by tjhei** [comment]
:+1: /run-tests

**Comment by bangerth** [comment]
Wasn't someone at the hackathon here working on this?

File: source/simulator/helper_functions.cc [hunk 1]
```diff
@@ -1112,7 +1112,7 @@ namespace aspect
   double
   Simulator<dim>::compute_initial_stokes_residual()
   {
-    LinearAlgebra::BlockVector remap (introspection.index_sets.stokes_partitioning, mpi_communicator);
+    LinearAlgebra::BlockVector zero_vel_sol (introspection.index_sets.stokes_partitioning, mpi_communicator);
```
Comment (bangerth) [inline source/simulator/helper_functions.cc]: I'm not fond of this variable name either. Could you spell it out completely? It's also not clear to me what the "zero" part is supposed mean. Could you maybe add a comment what the variable is supposed to be in the first place?

File: source/simulator/solver.cc [hunk 2]
```diff
@@ -659,26 +659,26 @@ namespace aspect
     // create a completely distributed vector that will be used for
     // the scaled and denormalized solution and later used as a
     // starting guess for the linear solver
-    LinearAlgebra::BlockVector remap (introspection.index_sets.stokes_partitioning, mpi_communicator);
+    LinearAlgebra::BlockVector init_sol (introspection.index_sets.stokes_partitioning, mpi_communicator);
```
Comment (bangerth) [inline source/simulator/solver.cc]: same here -- can you spell out what `init_sol` should be? initial? initialized? I guess we just like long variable names that are self-descriptive.

**Comment by bangerth** [comment]
Let's go with #1531 given that @alarshi is a newcomer :-)

**Pull request closed without merging**